### DISCUSSION TITLE
Can't access WordPress on different device in same network

### DISCUSSION THREAD

**Discussion by MattE-BB** [leokhoa/laragon#544] [Q&A]
Hi,

I would like to access my site using different devices in the same subnet to check for responsiveness on different physical devices, but am having lots of trouble. 

When I access the site on different devices the website doesn't render correctly (doesn't look like it's applying any CSS). I think this is due to installing WordPress using the quick app feature being associated with localhost and not an IP address. But, I can't see anyway to change this, or specify an IP address instead of localhost when I install WordPress using quick app. I'm still learning, so apologies if I'm not being clear, but any help would be really appreciated!

**Comment by almahmudbd** [comment]
I am facing the same problem here, can't access properly from another device in same lan.
ngrok not working at all.

**Comment by Vincent-HD** [comment]
This is because, laragon setup hostnames inside your `hosts` file, when you add an app, it point `<name_of_your_app>.test` -> `127.0.0.1` so when you try to access the site through the ip of your machine, laragon don't know what site you want to access, because it only sees that you are trying to access a site called <ip_of_your_machine> and not a hostname. to solve this.

I think you can add a entry inside your `hosts` file on the other PC, it should work after.
You can also try to setup (it's easy) a cloudflare tunnel, called cloudflared, it's the same as ngrok, but quicker, and you can access it outside of your home network, you can even bind it to a custom real domain

**Reply by jauhari** [reply]
I tried using cloudflared but why, this cloudflared goes to Default Website? and didn't go to my Virtual Host?

**Comment by LebToki** [comment]
Guys
check this thread [here](https://github.com/leokhoa/laragon/discussions/540) for a possible solutions

**Reply by MattE-BB** [reply]
Hi LebToki,

Apologies about the delayed response, I didn't notice that there was a reply. I've read through the thread and it seems very promising - I will try it tonight. Hopefully I can move back to Laragon from XAMPP. Thanks for pointing me in the right direction.

Thanks.

**Reply by LebToki** [reply]
Happy to have helped and yes I have tried them all and I love Laragon as is. Do check my index page package / landing page project and hope you like it

**Comment by MGeurts** [comment]
Just create a DNS record inside your LAN.
See [https://www.kreaweb.be/laravel-10-vite-configuration/](https://www.kreaweb.be/laravel-10-vite-configuration/).
